nx_main/main.c

```diff
@@ -40,7 +40,7 @@ int main(int argc, char **argv)
 
     rc = plInitialize();
     if (R_FAILED(rc)) fatalSimple(-6);
-
+    
```

Comment:
revert

Reply:
im confused? you want me to revert the newline? or the tab? this tab is automatically added when i press enter....

Reply:
Just remove the whitespace.